packages/orbit-components/src/CarrierLogo/index.tsx

```diff
@@ -98,11 +98,9 @@ export const StyledCarrierLogo = styled.div<{
   ${({ theme, carriers, size, inlineStacked }) => css`
     background-color: ${theme.orbit.backgroundCarrierLogo};
     height: ${carriers.length > 1 && !inlineStacked
-      ? theme.orbit.heightIconLarge
-      : `${getRenderSize({ theme, size })}px`};
-    width: ${carriers.length > 1
-      ? theme.orbit.widthIconLarge
+      ? theme.orbit.spaceXLarge
```

Comment:
Should we be using spacing tokens to define the height of elements? Sounds odd to me

Reply:
right, created the token for CarrierLogo